composer.json

```diff
@@ -8,7 +8,7 @@
         "source": "https://github.com/bowphp/framework"
     },
     "require": {
-        "php": ">=7",
+        "php": "^7.1",
```

Comment:
Require `^7.1` version at least to prevent syntax error on line [136](https://github.com/bowphp/framework/blob/4.0/src/Storage/Service/FTPService.php#L136) in `src/Storage/Service/FTPService.php` class.